src/Bicep.Core/TypeSystem/Az/OciTypeLoader.cs

```diff
@@ -33,7 +33,7 @@ public static OciAzTypeLoader FromTgz(string pathToGzip)
                 if (entry.DataStream is null)
                 {
                     var errorMessage = $"Failed to restore {entry.Name} from OCI provider data";
-                    throw new ArgumentException(errorMessage, nameof(entry.Name));
+                    throw new ArgumentException(errorMessage, entry.Name);
```

Comment:
[nit] I'd probably go with `InvalidOperationException` - "argument" here generally refers specifically to a C# function argument.